support/run-all-llms.sh

```diff
@@ -0,0 +1,124 @@
+echo
```

Comment:
It might be worth putting a little header in this so it's clear it's auto-generated.